locksmith/src/controllers/v2/lockSettingContoller.ts

```diff
@@ -0,0 +1,61 @@
+import * as z from 'zod'
+import { Request, Response } from 'express'
+import Normalizer from '../../utils/normalizer'
+import logger from '../../logger'
+import * as lockSettingOperations from '../../operations/lockSettingOperations'
+
+const LockSettingSchema = z.object({
+  sendEmail: z
+    .boolean({
+      description: 'When true enable to send emails',
+    })
+    .default(true),
+})
+
+export class LockSettingController {
+  async updateSettings(request: Request, response: Response) {
+    try {
+      const lockAddress = Normalizer.ethereumAddress(request.params.lockAddress)
```

Comment:
Let's just keep it consistent with latest pattern we use `const handler: RequestHandler = ()`

Reply:
Updated!